java/connector-node/README.md

```diff
@@ -73,6 +74,24 @@ Or you can use conda and install the necessary package `grpcio grpcio-tools psyc
 
 The connector service is the server and Python integration test is a client, which will send gRPC request and get response from the connector server. So when running integration_tests, remember to launch the connector service in advance. You can get the gRPC response and check messages or errors in client part. And check the detailed exception information on server side.
 
+### JDBC test
+
+We have integration tests that involve the use of several sinks, including file sink, jdbc sink, iceberg sink, and deltalake sink. If you wish to run these tests locally, you will need to configure both MinIO and PostgreSQL. 
+Downloading and launching MinIO is a straightforward process. For PostgreSQL, I recommend launching it using Docker. When setting up PostgreSQL, please ensure that the values for `POSTGRES_PASSWORD`, `POSTGRES_DB`, and `POSTGRES_USER` match the corresponding settings in the `integration_tests.py` file.
+
+```shell
+docker run --name my-postgres -e POSTGRES_PASSWORD=connector -e POSTGRES_DB=test -e POSTGRES_USER=test -d -p 5432:5432 postgres
```

Comment:
We may include a sample SQL that can create a sink to the pg instance launched from this command.

Reply:
You mean like [this](https://www.risingwave.dev/docs/current/sink-to-postgres/#sink-from-risingwave)?

Reply:
yes.